### PR TITLE
Add optional feature to upload PNG as badge background

### LINKED ISSUE #7: Allow users to upload a PNG for Badge background
Current UI only accepts a CSV file and uses certain predefined images. We should allow users to upload an image themselves

### PR DESCRIPTION
**Pull request by jajodiaraghav**
This PR allows user to upload an image which can be used as a background for badges. Uploading image is optional.
Fixes #7

### CODE REVIEW

File: app/templates/index.html [hunk 1]
```diff
@@ -19,6 +19,8 @@
 			    <div class="form-group">
 					<label for="inputFile">Upload a CSV file to generate badges</label>
 					<input type="file" id="inputFile" name="file">
+					<label for="imageFile">[Optional] Upload a PNG file for badge background</label>
+					<input type="file" id="imageFile" name="image">
```
Comment (niccokunzmann) [inline app/templates/index.html]: Here it  is "imageFile"

File: app/main.py [hunk 2]
```diff
@@ -26,14 +26,27 @@ def upload():
...
-    filename = file.filename
+    filename = file.filename    
+
     # if user does not select file, browser submits an empty part without filename
     if filename == '':
         flash('Please select a CSV file to Upload!', 'error')
         return redirect(url_for('index'))
 
+    # If a PNG is uploaded, push it to the folder
+    if request.files['image'].filename != '':
```
Comment (niccokunzmann) [inline app/main.py]: Here it is image.

File: app/templates/index.html (hunk 1 above)
Comment (niccokunzmann) [reply app/templates/index.html]: Does this work, then?

**Comment by niccokunzmann** [comment]
There are conflicts so I can not merge.

File: app/main.py (hunk 2 above)
Comment (jajodiaraghav) [reply app/main.py]: Hi Nicco, The 'imageFile' is the ID, but the request data is accessed using the "name" attribute.
Fixing conflicts in a minute

**Comment by niccokunzmann** [comment]
Thanks.

**Pull request merged**